tox.ini

```diff
@@ -0,0 +1,59 @@
+[tox]
+skipsdist = True
+usedevelop = True
+skip_missing_interpreters = True
+
+envlist = 
+    python{3.7,3.8,3.9}-django{3.0,3.1,3.2}-wagtail{2.15}
```

Comment:
Django 3.0 & 3.1 are no longer supported, should these be removed? These versions are also referenced in `classifiers` in `setup.py`.

Reply:
But wagtail 2.15 still can work with those django versions

https://docs.wagtail.org/en/stable/releases/upgrading.html#compatible-django-python-versions

Reply:
I would push for 3.2 being the min available one (as it is a version supported by Wagtail 2.15)

The reason for that is.. well, let's help people be on supported versions

Reply:
Updated to remove Django 3.0 & 3.1